dev/com.ibm.ws.jaxrs.2.1.common/src/com/ibm/ws/jaxrs21/sse/LibertySseEventSinkImpl.java

```diff
@@ -134,9 +144,17 @@ public boolean isClosed() {
             if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
                 Tr.debug(tc, "SseEventSink is closed - failed sending event:  " + event);
             }
-                throw new IllegalStateException("SseEventSink is closed.");  
```

Comment:
Rather than removing the indent on this line shouldn't the if statement above it be indented to match?

Reply:
I let Eclipse handle the indenting using the Source -> Correct Indentation, but it looks right to me now.